assets/src/web-stories-block/block/components/storiesInspectorControls.js

```diff
@@ -162,6 +184,10 @@ const StoriesInspectorControls = (props) => {
                       });
                     }
                   }}
+                  help={
+                    'archive_link' === field &&
+                    show && <RawHTML>{archiveLink}</RawHTML>
```

Comment:
Should be using a React component, not `RawHTML`

Reply:
```suggestion
                    show && < ArchiveLink  />
```